data/_models/shared/Contacts.csv

```diff
@@ -1,16 +1,17 @@
-tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible
-Contacts,,,,,,,,,,,vcard:Individual,Listing of contact persons per resource,"DataCatalogueFlat,EMA",
+tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible,computed
+Contacts,,,,,,,,,,,"vcard:Individual, foaf:Agent",Listing of contact persons per resource,"DataCatalogueFlat,EMA",
 Contacts,,collection,ref,1,true,,Collections,,,,,Resource the contact is affiliated with,"DataCatalogueFlat,EMA",
```

Comment:
dct:title?

Reply:
`vcard:Kind` does not really specify this, but `dct:title` would be confusing in my view

Reply:
but maybe we can use this to map back to the collection the contact belongs to? otherwise there is no link.